airflow/providers/microsoft/azure/hooks/cosmos.py

```diff
@@ -67,7 +67,7 @@ def get_connection_form_widgets() -> Dict[str, Any]:
         }
 
     @staticmethod
-    def get_ui_field_behaviour() -> Dict[str, Any]:
+    def get_ui_field_behaviour() -> Dict:
```

Comment:
Why changing this?

Reply:
Frankly that was not a deliberate change and I suspect my IDE may have tried to "help" me out there. I've just put it back to the way it was - thanks!